bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/parallacticangle/PositionAnglePanel.scala

```diff
@@ -221,15 +217,26 @@ class PositionAnglePanel[I <: SPInstObsComp with PosAngleConstraintAware,
   }
 
   /**
+    * The actual copying of a given pos angle to the data object.
+    * This is done explicitly to avoid overwriting based on minor differences in double precision and to avoid
+    * adding 180 to the position angle, which causes issues.
+    **/
+  private def setInstPosAngle(newAngleDegrees: Double): Unit = editor.foreach { e =>
+    val Precision = 0.005
+    val oldAngleDegrees = e.getDataObject.getPosAngle
+    if (Math.abs(oldAngleDegrees - newAngleDegrees) >= Precision && Math.abs(Math.abs(oldAngleDegrees - newAngleDegrees) - 180) >= Precision) {
+      e.getDataObject.setPosAngle(newAngleDegrees)
```

Comment:
I still don't quite get what is going on here.  What is sequence of events with the 180 flip that is problematic?  If we've chosen to observe at the average parallactic angle, is the position angle still user editable?  If so why?